og_access/og_access.install

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * @file
+ * Install/update/uninstall hook implementations.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function og_access_install() {
+  // Rebuild node access permissions.
+  node_access_rebuild(TRUE);
```

Comment:
```suggestion
  node_access_needs_rebuild(TRUE);
```

Reply:
with a comment of `Mark node access permissions for rebuild.` or something along the lines

Reply:
whoops :-)